src/Kafka/CreateKafkaInstance/components/ModalAlerts.tsx

```diff
@@ -16,89 +21,89 @@ export const ModalAlerts: VoidFunctionComponent<ModalAlertsProps> = ({
 }) => {
   const { t } = useTranslation("create-kafka-instance");
 
-  switch (true) {
-    case isLoading:
-      return (
-        <Alert
-          id="mk-create-instance-quota-alert"
-          className="pf-u-mb-md"
-          variant={AlertVariant.info}
-          title={t("checking_instance_title")}
-          aria-live="polite"
-          isInline
-          customIcon={
-            <Spinner
-              size="md"
-              aria-valuetext={t("checking_instance_message")}
-            />
-          }
-        />
-      );
-    case isSystemUnavailable:
-      return (
-        <Alert
-          id="mk-create-instance-quota-alert"
-          className="pf-u-mb-md"
-          variant={AlertVariant.warning}
-          title={t("modal_alerts.system_unavailable_title")}
-          aria-live="polite"
-          isInline
-        >
-          {t("modal_alerts.system_unavailable_message")}
-        </Alert>
-      );
-    case instanceAvailability === "trial":
-      return (
-        <Alert
-          id="mk-create-instance-quota-alert"
-          className="pf-u-mb-md"
-          variant={AlertVariant.info}
-          title={t("modal_alerts.trial_available_title")}
-          aria-live="polite"
-          isInline
-        />
-      );
-    case instanceAvailability === "over-quota":
-      return (
-        <Alert
-          id="mk-create-instance-quota-alert"
-          className="pf-u-mb-md"
-          variant={AlertVariant.warning}
-          title={t("modal_alerts.over_quota_title")}
-          aria-live="polite"
-          isInline
-        >
-          {t("modal_alerts.over_quota_message")}
-        </Alert>
-      );
-    case instanceAvailability === "trial-used":
-      return (
-        <Alert
-          id="mk-create-instance-quota-alert"
-          className="pf-u-mb-md"
-          variant={AlertVariant.warning}
-          title={t("modal_alerts.trial_used_title")}
-          aria-live="polite"
-          isInline
-        >
-          {t("modal_alerts.trial_used_message")}
-        </Alert>
-      );
-    case instanceAvailability === "trial-unavailable":
-    case instanceAvailability === "regions-unavailable":
-    case instanceAvailability === "instance-unavailable":
-      return (
-        <Alert
-          id="mk-create-instance-quota-alert"
-          className="pf-u-mb-md"
-          variant={AlertVariant.warning}
-          title={t("modal_alerts.instance_or_region_unavailable_title")}
-          aria-live="polite"
-          isInline
-        >
-          {t("modal_alerts.instance_or_region_unavailable_message")}
-        </Alert>
-      );
-  }
-  return null;
+  return (
+    <AlertGroup>
+      {(() => {
```

Comment:
Odd syntax, I know. I didn't really want to create a component just to wrap the AlertGroup. But I had to add the AlertGroup to address the a11y hints from PatternFly about how to make alerts accessible.